src/ccstudiodss/api.py

```diff
@@ -77,7 +78,7 @@ def __enter__(self):
             self.debug_server = self.script.getServer("DebugServer.1")
             self.debug_server.setConfig(ccstudiodss.utils.fspath(self.ccxml))
 
-            self.debug_session = self.debug_server.openSession()
+            self.debug_session = self.debug_server.openSession(self.device_string)
```

Comment:
```suggestion
            self.debug_session = self.debug_server.openSession(sPattern=self.device_string)
```

I don't recall offhand but if this works with the parameter name specified that would be preferred.  If not, no biggy.  I just don't know what `javabridge` allows here.

Reply:
Tried `sPattern=self.device_string`, didn't work. But just `self.device_string` does